src/FormattedText.tsx

```diff
@@ -62,7 +62,7 @@ const renderMarkdown = (
           // convert <br> tags to blank lines for markdown
           .replace(/<br\s*\/?>/ig, '\n')
           // URL encode all links
-          .replace(/\[(.*?)\]\((.*?)( +".*?"){0,1}\)/ig, (match, text, url, title) => `[${text}](${markdownIt.normalizeLink(url)}${title === undefined ? '' : title})`);
+          .replace(/\[(.*?)\]\((.*?)( +".*?"){0,1}\)/ig, (match, text, url, title) => `[${text}](${markdownIt.normalizeLink(url)}${title === undefined ? '' : title}){:target=${window.location.hostname === url.hostname || url.hostname.endsWith(`.${window.location.hostname}`) ? "_self" : "_blank"}}`);
```

Comment:
Tohle mi tbh přijde jak totální porno, a ne v tom pozitivním smyslu :D Ono teda už před tou úpravou, ale teď tím spís. 

Ten řádek má 290 znaků, a musel jsem na něj chvíli civět, než mi došlo, co se tu vlastně něm děje. Mně by rozhodně dávalo smysl to vyčlenit někam do funkce a dobře pojmenovat ji, i ty mezistavy (nebo funkce co je vytvářej).

To, že nad tím řádkem musí bejt koment, kterej vysvětluje, co ten řádek dělá, je pro mě velká indikace toho, že je něco špatně.

Co si o tom myslíš Ty?

Reply:
Hele, mně tohle nepřijde o moc čitelnější. Udělal jsem proposal toho, jak by mi to dávalo smysl - měl bys čas a chuť si zavolat a popovídat si o tom? :)